src/blocks/renderers/customForm.tsx

```diff
@@ -286,6 +288,41 @@ export const customFormRendererSchema = {
   required: ["schema"],
 };
 
+// Initialize all empty string fields to empty string
```

Comment:
I think "string fields without a default" is a little more clear here than just "empty string fields"

```suggestion
// Initialize all string fields without a default to empty string
```

Reply:
The problem is that RJSF doesn't update the string fields if the property is not in the data. Thus need to initialize all `null` and `undefined` string fields without a default value.